acds/archetypes/ron.py

```diff
@@ -118,7 +122,7 @@ def cell(
         """
         hz = hz + self.dt * (
             torch.tanh(
-                torch.matmul(x, self.x2h) + torch.matmul(hy, self.h2h) + self.bias
+                torch.matmul(x, self.x2h) + torch.matmul(hy, self.h2h - self.diffusive_matrix) + self.bias
```

Comment:
the diffusive term is applied regardless of the topology. this changes the behavior of previous implementations. Perhaps you could set the diffusive term to zero by default and set it only for the antisymmetric matrix. In this way the code stays compatible with previous versions of RON.

The same applies also for the changes in the other model classes.